.github/actions/prepare-node/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Prepare node environment
+runs:
+  using: "composite"
+  steps:
+    - name: Prepare Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14.x
+        cache: 'yarn'
+        path: node_modules
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
```

Comment:
These two properties `path` and `key` are **not** valid for `actions/setup-node`.